bin/init.sh

```diff
@@ -1,6 +1,13 @@
 #!/bin/bash
 set -ex
 
+# Ensure expected GOPATH setup
+PDIR=`pwd`
+if [ $PDIR != "$GOPATH/src/istio.io/pilot" ]; then
```

Comment:
I should have caught this but this is incorrect. GOPATH doesn't need to be set (defaults to ~/go https://golang.org/cmd/go/#hdr-GOPATH_environment_variable )

Reply:
fixing in https://github.com/istio/pilot/pull/1181